input_topic: "/camera/depth/color/points"
output_topic: "/segmented_cloud"

# Debug mode parameters
debug_mode: true # if true, it will process each input message one by one, waiting for user input on the CLI
queue_size: 40 # the number of input messages to be loaded and processed one by one
publish_supervoxel_cloud: true # if true, it will publish the supervoxel cloud intermediate step
supervoxel_cloud_topic: "/supervoxel_cloud" # topic to publish the supervoxel cloud
centroid_marker_topic: "/centroid_markers" # topic to publish the centroid marker

# Downsampling parameters
voxel_leaf_size: 0.005 

# Depth filter parameters - z-axis
min_depth: 0.1
max_depth: 5.0

# LCCP segmentation parameters
voxel_resolution: 0.01
seed_resolution: 0.02
color_importance: 0.1
spatial_importance: 1.0
normal_importance: 4.0
use_supervoxel_refinement: true
supervoxel_refinement_iterations: 3

concavity_tolerance_threshold: 20.0
k_factor: 0.0
min_segment_size: 20
use_smoothness_check: true
smoothness_threshold: 1
small_segment_threshold_percent: 0.2
use_single_camera_transform: true